### PR TITLE
w-13216730 convert to valkyr

### PR DESCRIPTION
**Pull request by HONTAKYU**

### CODE REVIEW

File: valkyr.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+product: pce
```
Comment (diegostrubolini) [inline valkyr.yaml:1]: I think by default the slackChannel will be the product + "-bot" so I think we need to override it: https://github.com/mulesoft/onprem-config-api/blob/master/valkyr.yaml#L9

File: images/hook/Dockerfile [hunk 2]
```diff
@@ -1,16 +1,20 @@
-FROM quay.io/gravitational/debian-grande:stretch
+FROM artifacts.msap.io/mulesoft/core-paas-base-image-ubuntu:v5.2.24@sha256:0be2945940a7a7ee25cc978a629d1dbce039050eb4f140a637713894cadca87a
```
Comment (diegostrubolini) [inline images/hook/Dockerfile:1]: Any reason not to point to the latest?

**Review by diegostrubolini — CHANGES_REQUESTED** [review]

File: valkyr.yaml (hunk 1 above)
Comment (HONTAKYU) [reply valkyr.yaml:1]: Done

File: images/hook/Dockerfile (hunk 2 above)
Comment (HONTAKYU) [reply images/hook/Dockerfile]: I think it's a while since this change was made, updated to the latest

**Review by fberckemeyer — COMMENTED** [review]
@HONTAKYU, it would be useful to see a passing anypoint build that uses the code from this PR before the PR gets merged. This is the standard pattern for any service we use in anypoint.

So for example ...
https://jenkins.build.msap.io/job/PCE/job/anypoint-components/job/gravity-apps/job/cluster-ssl-app-new/job/PR-8/7/artifact/valkyr-manifest.yaml/*view*/
... update the spec in anypoint to use 4.0.0-2.w-13216730, regenerate the manifest and try to get a passing build.

File: Makefile [hunk 3]
```diff
@@ -5,6 +5,8 @@ endif
```
Comment (alejandrorascovan) [inline Makefile:5]: Who's calling the Makefile now that the repo is built with Valkyr?

**Comment by HONTAKYU** [comment]
> @HONTAKYU, it would be useful to see a passing anypoint build that uses the code from this PR before the PR gets merged. This is the standard pattern for any service we use in anypoint.
> 
> So for example ... https://jenkins.build.msap.io/job/PCE/job/anypoint-components/job/gravity-apps/job/cluster-ssl-app-new/job/PR-8/7/artifact/valkyr-manifest.yaml/*view*/ ... update the spec in anypoint to use 4.0.0-2.w-13216730, regenerate the manifest and try to get a passing build.

Here we go: https://jenkins.build.msap.io/job/PCE/job/anypoint/job/PR-3820/17

File: Makefile (hunk 3 above)
Comment (HONTAKYU) [reply Makefile]: Removed

**Pull request merged**